### ISSUE TITLE
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

### ISSUE THREAD

**Issue by gaodabei** [multimodal-art-projection/YuE#146]
RTX 6000 ada

**Comment by gaodabei** [comment]
I'm really looking forward to someone helping me resolve this issue—it would make running the project go very smoothly for me.
